README_PYPI.rst

```diff
@@ -1,50 +1,92 @@
-in the industry including DMTF management standards and  the `SNIA`_
-Storage Management Initiative Specification `SMI-S`_.
+CIM/WBEM standards are used for a wide variety of systems management tasks
+in the industry including DMTF management standards and the `SNIA`_
+Storage Management Initiative Specification (`SMI-S`_).
 
-Pywbemcli provides the functionality to :
+Pywbemcli provides access to WBEM servers from the command line.
+It provides functionality to:
 
-* Explore the CIM data of WBEM Servers.  It can manage/inspect the CIM
-  model components including CIM classes, CIM instances, and CIM
-  qualifiers and execute CIM methods and queries on the WBEM server. It
-  implements subcommands to execute the CIM-XML operations defined in the
-  DMTF specification `CIM Operations Over HTTP(DSP0200)`_
+* Explore the CIM data of WBEM servers. It can manage/inspect the CIM model
+  components including CIM classes, CIM instances, and CIM qualifiers and
+  execute CIM methods and queries on the WBEM server.
 
-* Inspect/manage WBEM server functionality such as CIM namespaces, WBEM
-  registered profiles, and other server information.
+* Execute specific CIM-XML operations on the WBEM server as defined in `DMTF`_
+  standard `DSP0200 (CIM Operations over HTTP)`_.
 
-* Capture detailed information on interactions with the WBEM server
-  including time statistics.
+* Inspect and manage WBEM server functionality including:
 
-For more information on pywbemtools see the `pywbemtools readme`_ and the
-`pywbemtools documentation`_
+  * CIM namespaces
+  * Advertised WBEM management profiles
+  * WBEM server brand and version information
 
-.. _pywbemtools readme: https://github.com/pywbem/pywbemtools/blob/master/README.rst
-.. _pywbemtools documentation: https://pywbemtools.readthedocs.io/en/stable/
+* Capture detailed information on CIM-XML interactions with the WBEM server
+  including time statistics and details of data flow.
 
-For more information on pywbem, see the `pywbem readme`_, and the
-`pywbem documentation`_.
+* Maintain a file with persisted WBEM connection definitions so that pywbemcli
+  can access multiple WBEM servers by name.
 
-.. _pywbem: https://github.com/pywbem/pywbem
-.. _pywbem readme: https://github.com/pywbem/pywbem/blob/master/README.rst
-.. _pywbem documentation: https://pywbem.readthedocs.io/en/stable/
+* Provide both a command line mode and an interactive mode where multiple
+  pywbemcli commands can be executed within the context of a WBEM server.
 
-.. _CIM/WBEM standards: https://www.dmtf.org/standards/wbem/
+* Use an integrated mock WBEM server to try out commands. The mock server
+  can be loaded with CIM objects defined in MOF files or via Python scripts.
+
+
+Installation
+------------
+
+Requirements:
+
+1. Python 2.7, 3.4 and higher
+
+2. Operating Systems: Linux, OS-X, native Windows, UNIX-like environments on
+   Windows (e.g. Cygwin)
+
+The quick way for Linux and OS-X:
+
+.. code-block:: bash
```

Comment:
Again, we the .sh is ours because of cygwin and we wget the .bat

Reply:
See previous comment.